lib/internal/validators.js

```diff
@@ -34,10 +34,20 @@ const {
 } = require('internal/util/types');
 const { signals } = internalBinding('constants').os;
 
+/**
+ * @callback isInt32
+ * @param {number} value
+ * @returns {boolean}
+ */
 function isInt32(value) {
   return value === (value | 0);
 }
 
+/**
+ * @callback isUint32
+ * @param {number} value
+ * @returns {boolean}
+ */
```

Comment:
```suggestion
 */
/** @type {isUint32} */
```

Reply:
This change throws an error on WebStorm

<img width="755" alt="Screen Shot 2022-08-17 at 9 46 49 AM" src="https://user-images.githubusercontent.com/1935246/185150348-ebc0ff6d-9fd7-48ff-806b-4e9451e11033.png">

Reply:
Oh of course, it's not using `hideStackTrace`.